package/src/components/Attachment/AudioAttachment.tsx

```diff
@@ -328,35 +329,29 @@ const AudioAttachmentUploadPreviewWithContext = <
   );
 };
 
-export type AudioAttachmentUploadPreviewProps<
+export type AudioAttachmentProps<
   StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics,
-> = Partial<AudioAttachmentUploadPreviewPropsWithContext<StreamChatGenerics>> & {
+> = Partial<AudioAttachmentPropsWithContext<StreamChatGenerics>> & {
   index: number;
-  item: FileUpload;
+  item: Omit<FileUpload, 'state'>;
   onLoad: (index: string, duration: number) => void;
   onPlayPause: (index: string, status?: boolean) => void;
```

Comment:
```suggestion
  onPlayPause: (index: string, currentlyPlayed?: boolean) => void;
```

the status seems too generic here and non understandable.

Reply:
Fixed, looking for your final review and merge! Thanks 😄